src/components/fields/textarea/textarea.tsx

```diff
@@ -89,19 +89,22 @@ export const Textarea = (props: IGenericFieldProps<ITextareaSchema>) => {
 		<Form.CustomField label={label} id={id}>
 			<Wrapper chipPosition={chipPosition}>
 				{renderChips()}
-				<StyledTextarea
-					{...otherSchema}
-					{...otherProps}
-					id={id}
-					data-testid={TestHelper.generateId(id, "textarea")}
-					name={name}
-					maxLength={maxLength}
-					rows={rows}
-					resizable={resizable}
-					onChange={handleChange}
-					value={stateValue}
-					errorMessage={error?.message}
-				/>
+				<div aria-invalid={!!error?.message}>
```

Comment:
you can take in the latest changes from master. aria-invalid is now applied on the element

Reply:
rebased onto master, no longer manually adding div with aria-invalid attribute